recipes/cubicinterpolation/all/patches/fix-msvc.patch

```diff
@@ -0,0 +1,34 @@
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
```

Comment:
Since you're the author of the library, may I ask you to add these changes directly to your project, release a new version and create a recipe for an updated version?

This way we will get rid of this patch, which might break in the future.
And, probably more important, your library will be easily usable in MSVC, which is be a good thing (and not just through conan).

Reply:
It might be too much, and, if you're not ready to go an extra mile, tell me, and I will approve current changes (I don't want to stop you from sharing your lib through conan).

Reply:
This is no problem at all and also my favorite solution. :)